demo/js/index.js

```diff
@@ -89,12 +89,13 @@ function filterBreadcrumbActions(action) {
   return !noisy;
 }
 
-Raven.config(SENTRY_DSN, {
+Sentry.init({
+  dsn: SENTRY_DSN,
   /* global COMMITHASH */
   release: typeof COMMITHASH !== "undefined" ? COMMITHASH : "DEV",
-}).install();
+});
 
-const ravenMiddleware = createMiddleware(Raven, {
+const ravenMiddleware = createMiddleware(Sentry, {
```

Comment:
nit: Should this variable and it's references be updated to `sentryMiddleware`?

Reply:
Ah thanks. I'll update.